src/components/download-upload-history/download-upload-history-button.tsx

```diff
@@ -1,6 +1,10 @@
 import React, { Component } from "react";
 
-class DownloadUploadHistoryButton extends Component {
+interface IDownloadUploadHistoryButton {
```

Comment:
since the other one is called `IDownloadFormSlideProps`, let's stick to the same naming pattern: `IDownloadUploadHistoryButtonProps`